### ISSUE TITLE
[Suggestion] The ability to use Homebrew for installation

### ISSUE THREAD

**Issue by RinMinase** [Marekkon5/onetagger#73]
Updating the package via the terminal seems easier compared to:
- opening the browser / email (watching for releases)
- checking the repository for new releases (or watching for releases)
- downloading the new package
- removing the previously installed package
- installing the downloaded package
- removing the downloaded package

This is gust a suggestion though. 😊
I am not sure of the complexity of creating casks or taps in Homebrew.

**Comment by Marekkon5** [comment]
I fully agree with that, however I don't have a Mac, never used it, so I have no idea how to make a package, but I can try looking into it. 

BTW: There is also a built in updater directly in 1T.

**Comment by RinMinase** [comment]
Ohhh. I see. 😮 

If there's a built in updater already in the app then it's fine already as it is. No need to complicate it further. I initially thought that the update process was the one I mentioned on top. 😁

Then the **slight advantage** of using the homebrew process would be for installation purposes only.

You can close this ticket if you want as the update process is already fine as it is. Not closing this myself since this ticket may be your tracker or TODO for the homebrew process, if you so want to continue on it. 😊
